docs.md

```diff
@@ -1,23 +1,499 @@
+allow {
+   input.request_path == ["v1", "collections", "lakes", ""]
+   input.groupname == "admin"
+}
+```
+   ### input_props_in
+   This logic checks if a particular property on the input object is present in a list of values from the database. <br />
+
+   Example:
+
+   ```json
+   {
+      "command": "input_prop_in",
+      "properties": {
+         "input_property": "company",
+         "datasource_name": "items",
+         "datasource_loop_variable": "name"
+      }
+}
+   ```
+   In the json above, the `input_property` key holds the property that is to be validated before the request to that path is passed. The `datasource_name` key holds the name of the datasource(a list) from the database. The `datasource_loop_variable` key holds the name of the key on each object in the datasource.
+
+   The rego rules combine data from the database with the input object, to work out certain conditions. The datasource is the list of values from the database, and the datasource_loop_variable is the name of the key on each object in the datasource.
+
+   The REGO equivalent of the above rule object is: <br />
+   ```rego
+   `input.company == data.items[i].name`
+   ```
+   ### input_props_in_as
+   This logic checks if the value of two properties on the input object is present on one object in the database <br />
+
+   Example:
+
+   ```json
+ {
+      "command": "input_prop_in_as",
+      "properties": {
+         "datasource_name": "items",
+         "datasource_loop_variables": ["name", "everyone"],
+         "input_properties": ["preferred_username", "groupname"]
+      }
+ }
+   ```
+   In the json above, the resulting REGO code loops over the datasource twice, checking for equality between the values of the input properties, and the values of the datasource loop variables.
+
+   The REGO equivalent of the above rule object is: <br />
+   ```rego
+      some i
+      data.items[i].name == input.preferred_username
+      data.items[i].everyone == input.groupname
+   ```
+
+
+### allow_full_access
+   This logic allows full access to the resources defined. If the value of the property on the input object has a particular value<br />
+
+   Example:
+   ```json
+   [
+      {
+         "command": "allow_full_access",
+         "properties": {
+            "input_property": "groupname",
+            "value": "EDITOR_ATAC",
+         },
+      }
+   ]
+   ```
+   In the JSON above, the result is an allow block that allows access to the resources defined if the value of the property on the input object has a particular value.
+
+   ```rego
+   allow {
+      input.groupname == "EDITOR_ATAC"
+   }
+   ```
+
+
+
+
+POST `/policies/`  Create new policy
+============
+Using the JSON defined from the frontend, this route is used to build a new policy. The response will be a REGO file written and pushed to GitHub as a newly established remote repository, with the request body conforming to a specified syntax described by the pydantic model `Policy`. <br />
```

Comment:
Again, we should not make it seem like only the frontend can send JSON. A better way to start this section:

The POST route takes a request body containing the rules defined according to the schema.